tests/test_datasets/test_top_down_dataset.py

```diff
@@ -28,19 +28,21 @@ def load_json_to_output(json_name, prefix=''):
 
 def convert_db_to_output(db):
     outputs = []
-
+    bbox_id = 0
     for item in db:
         keypoints = item['joints_3d'].reshape((1, -1, 3))
         center = item['center']
         scale = item['scale']
+        bbox_ids = [bbox_id]
```

Comment:
Do we really need list?

Reply:
Yes, it must be a list and the length of the list would equal to the batch size.